pallets/gear/src/tests.rs

```diff
@@ -2889,7 +2889,6 @@ fn test_two_contracts_composition_works() {
 //
 // Note: on manager level message will not be included to the [queue](https://github.com/gear-tech/gear/blob/master/pallets/gear/src/manager.rs#L351-L364)
 // But it's is not preferable to enter that `if` clause.
-// todo # 929 After create_program sys-call becomes fallible, tests must be changed
 #[test]
 fn test_create_program_with_value_lt_ed() {
```

Comment:
I think `test_create_program_with_value_lt_ed` and `test_create_program_with_exceeding_value` should result in different storage mutations after adding the fallible feature. For example, **no** mails should be sent to user after unsuccessful calls to `create_program_with_gas`, but these tests base on that behaviour

Reply:
So what do you want me to do?

Reply:
Would be nice if you test the fallible feature on one of tests I stated in the last comment. That requires: handling `Result` of the call in the program (not `unwrap`ing), checking whether mail with error code wasn't sent to the original message sender (so must have logically opposite to current mail existence check). Was it clear?